src/pages/components/NavContainer/Nav/Nav.js

```diff
@@ -59,18 +57,20 @@ class Nav extends Component {
                   key={idx}
                   className="menu"
                   onMouseOver={() => this.handleMenuStyle(idx)}
-                  // onClick={() => this.goToPage(idx)}
                 >
                   <small>{menu.koreanName}</small>
                   <span>{menu.englishName}</span>
                 </div>
               ))}
             </ul>
           </div>
-          <div className="userInfo">
-            <div className="userName">
+          <div className={userName !== null ? "userInfo" : "userInfo inactive"}>
```

Comment:
classname 동적사용 잘해주셨네요! 👏🏻
null은 boolean값이 false입니다. 아래와 같이 적어주실 수 있겠네요! 
```suggestion
          <div className={userName ? "userInfo" : "userInfo inactive"}>
```

Reply:
넵 신영님 수정했습니다!